### ISSUE TITLE
access to %SCIRUNDATA% path variable in python console

### ISSUE THREAD

**Issue by jessdtate** [SCIInstitute/SCIRun#1566]
### Description

Having access to the %SCIRUNDATA% path variable in python console and IWP would allow for importing python library dependency easier.  This could either be an environment variable, or a function call.   I think the environment variable would be better.  

### Steps to Reproduce

env variable
1. get variable of SCIRUNDATA path
2. use path to load in other files

function
1. call function 
2. return SCIRUNDATA path as a string.
3. use string to load other files.

**Comment by dcwhite** [comment]
I went with option 1, and just added it to `os.environ`:
```
>>> os.environ["SCIRUNDATADIR"]
'C:/_/SCIRunTestData'
```

**Comment by jessdtate** [comment]
This works as describe.  While I was working on this I noticed something else.  When I try to type a path out that is the same as the SCIRUNDATADIR path in the IWP module, it is automatically changed to %SCIRUNDATADIR%.  So '/Users/jess/software/BrainStimulator' changes to '%SCIRUNDATADIR%'.  If the path is already in the IWP script, an the datadir is changed in the preferences, the IWP script will automatically change ( I think only if something else in the script changes).  This less of a problem because of the new fix, but it is annoying and could be an issue is some cases.  I just noticed that It happens in all modules (even create matrix).
